src/renderers/dom/shared/__tests__/ReactMount-test.js

```diff
@@ -345,4 +349,45 @@ describe('ReactMount', () => {
     expect(container1.textContent).toEqual('2');
     expect(container2.textContent).toEqual('a!');
   });
+
+  if (ReactDOMFeatureFlags.useFiber) {
+    describe('mount point is a comment node', () => {
+      let containerDiv;
+      let mountPoint;
+
+      beforeEach(() => {
+        const ReactFeatureFlags = require('ReactFeatureFlags');
+        ReactFeatureFlags.disableNewFiberFeatures = false;
+
+        containerDiv = document.createElement('div');
+        containerDiv.innerHTML = 'A<!-- react-mount-point-unstable -->B';
+        mountPoint = containerDiv.childNodes[1];
+        invariant(mountPoint.nodeType === 8, 'Expected comment');
+      });
+
+      it('renders at a comment node', () => {
+        function Char(props) {
+          return props.children;
+        }
+        function list(chars) {
+          return chars.split('').map(c => <Char key={c}>{c}</Char>);
+        }
+
+        ReactDOM.render(list('aeiou'), mountPoint);
+        expect(containerDiv.innerHTML).toBe(
+          'Aaeiou<!-- react-mount-point-unstable -->B',
+        );
+
+        ReactDOM.render(list('yea'), mountPoint);
```

Comment:
Why have both this and the previous expectation? They seem to be testing the same thing.